src/pages/wallets/find-wallet.tsx

```diff
@@ -507,6 +545,98 @@ const FindWalletPage = ({ data, location }) => {
             )}
           </div>
         </FilterSidebar>
+        <FilterSidebarMobile
+          variants={{
+            active: {
+              x: 0,
+            },
+            inactive: {
+              x: -350,
+            },
+          }}
+          animate={controls}
+          showMobileSidebar={showMobileSidebar}
+          initial="inactive"
+          transition={{
+            type: "tween",
+            duration: 0.2,
+          }}
+        >
+          <FilterTabs>
```

Comment:
Theres a lot of duplicate code here. This could be extracted to a component to clean this up instead of duplicating.